include/apps/watchfaces/OswAppWatchfaceDigital.h

```diff
@@ -13,9 +13,13 @@ class OswAppWatchfaceDigital : public OswApp {
   virtual void loop() override;
   virtual void stop() override;
   ~OswAppWatchfaceDigital(){};
+  static uint8_t getDateFormat();  // Return 0 : mm/dd/yyyy 1 : dd.mm.yyyy 2 : yy.mm/dd
+  static void drawSteps();
+  static void digitalWatch(short timeZone, uint8_t fontSize, uint8_t dateCoordY, uint8_t timeCoordY);
```

Comment:
I want to do re-use, but if I turn over the position and font, I will give at least 7 parameters. Can I change this?

Reply:
Maybe you could declare a parameter-struct as needed. E.g. the Vulkan SDK for C++ does this exclusively! Otherwise just go ahead...